pullrequest.rst

```diff
@@ -121,25 +121,28 @@ Submitting
 
 Once you are satisfied with your work, you will need to commit your
 changes to your branch. Assume you want to add a file which you have changed,
-you can add it to the list of changes to be committed (the "staging area")::
+you will need to enter this command::
 
    git add <file>
 
-After adding all files of changed, you can commit your staged changes::
+After adding all files of changed, you can commit your changes::
 
    git commit
 
-Inside the commit editor, commit messages title should startswith
-``bpo-NNNNN`` and the major changed of this commit, for example::
+Inside the commit editor, commit message's title should start with
+``bpo-NNNNN`` and the major change of this commit, for example::
 
    bpo-29624: Adds purge step and layout test after uploading files.
 
-You will saw hash tag number in ``git log``, e.g.
-``bpo-29624: Adds purge step and layout test after uploading files. (#258)``
-, you won't need to add this into commit title, it will be auto added when
-making a pull request on GitHub.
+Two commands, ``git log`` and ``git status``, provide helpful information
+and detail about commits cand changes.
 
-You can always run ``git status`` to see what changes are outstanding.
+.. note::
+
+   You will saw hashtag with number in git log, e.g.
```

Comment:
should this be more specific to "... that references the pull request id in cpython repo."